inference-engine/src/mkldnn_plugin/mkldnn_extension.cpp

```diff
@@ -10,6 +10,11 @@
 
 #include <ngraph/ngraph.hpp>
 #include <ngraph_ops/type_relaxed.hpp>
+#include <ngraph_ops/nms_ie_internal.hpp>
+#include <ngraph_ops/convolution_ie.hpp>
+#include <ngraph_ops/deconvolution_ie.hpp>
```

Comment:
AFAIK, we don't use convolution_ie and deconvolution_ie inside the plug-in, so they shouldn't appear during serialization.

Reply:
Ok, removed